### ISSUE TITLE
Unknown lvalue 'Author' in section 'Unit' (using systemd)

### ISSUE THREAD

**Issue by alamut** [bflad/chef-docker#314]
```
$ systemctl status web-ui-dev
● web-ui-dev.service - Docker service for web-ui-dev
   Loaded: loaded (/etc/systemd/system/web-ui-dev.service; enabled)
   Active: inactive (dead)

May 12 13:31:07 docker-node-1-aws-debian8 systemd[1]: [/etc/systemd/system/web-ui-dev.service:3] Unknown lvalue 'Author' in section 'Unit'
```
The service didn't start at instance's startup, of course.
Seems there's no such directive as `Author` at all http://www.freedesktop.org/software/systemd/man/systemd.unit.html

Using debian 8.0 (jessie).
Relevant for version 0.37.0 and master upstream, as far as I can see.

**Comment by alamut** [comment]
I also think the `WantedBy=local.target` directive is invalid for Jessie. I was unable to find such systemd targer.
I'm not really good in systemd, but as for me, there should be something like
```
WantedBy=multi-user.target
```
or
```
WantedBy=docker.service
```
Both works in my case.

**Comment by someara** [comment]
This should be fixed up as of 0.40.x... feel free to re-open if this is still an issue
